android/anc/src/main/java/org/smartregister/fhircore/anc/ui/report/ReportHomeScreen.kt

```diff
@@ -63,35 +56,19 @@ const val TOOLBAR_BACK_ARROW = "toolbarBackArrow"
 
 @Composable
 fun ReportHomeScreen(viewModel: ReportViewModel) {
-  Surface(color = colorResource(id = R.color.white)) {
-    Column {
 
-      // top bar
-      TopAppBar(
-        title = {
-          Text(text = stringResource(id = R.string.reports), Modifier.testTag(TOOLBAR_TITLE))
-        },
-        navigationIcon = {
-          IconButton(onClick = viewModel::onBackPress, Modifier.testTag(TOOLBAR_BACK_ARROW)) {
-            Icon(Icons.Filled.ArrowBack, contentDescription = "Back arrow")
-          }
-        }
-      )
-
-      val lazyReportItems = viewModel.getReportsTypeList().collectAsLazyPagingItems()
+  val lazyReportItems = viewModel.getReportsTypeList().collectAsLazyPagingItems()
 
-      LazyColumn(modifier = Modifier.background(Color.White).fillMaxSize()) {
-        itemsIndexed(lazyReportItems) { _, item -> ReportRow(item!!, { _, _ -> }) }
+  LazyColumn(modifier = Modifier.background(Color.White).fillMaxSize()) {
```

Comment:
This list should have a unique identifier to prevent bugs that may occur during recomposition. You can provide any unique property of the items as the key

Reply:
@ellykits done